python/tvm/relay/transform.py

```diff
@@ -1004,3 +1004,25 @@ def visit_var(self, var):
                 else:
                     return var
         return ChangeBatchMutator().visit(func)
+
+def EnableTrt(trt_version=(6, 0, 1)):
```

Comment:
Can we have a utility to find the installed trt version instead of being specified by users?

Reply:
Sure, I can add that as the default option. But this pass is meant to be used by the compiler on a system which doesn't need to have TRT installed which is why it needs to know which version to target for the partitioning.